docs/content/concepts/cryptography/passkeys.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Passkeys
+description: Passkeys is a Sui primitive that enables you to sign-in to apps and send transactions to Sui using a private key securely stored on a authenticator. It uses the WebAuthn standard.
+---
+
+Passkeys provide a secure and user-friendly alternative for submitting transactions to Sui. Built on the **WebAuthn standard**, passkeys let users authenticate and sign transactions using:
+
+- Hardware security keys, such as YubiKeys)
+- Mobile devices, such as smartphones and tablets
+- Platform-based authenticators, such as Face ID and Touch ID
+
+Passkeys simplify authentication by removing the need to manage seed phrases or private keys manually. Instead, they rely on device-based authentication and cloud synchronization, allowing seamless, phishing-resistant access across multiple devices.
+
+By integrating passkeys, Sui improves security and accessibility, making it easier for users to manage their accounts without compromising decentralization or cryptographic security.
+
+Refer to [Typescript SDK support](https://sdk.mystenlabs.com/typescript/cryptography/passkey) on how to add passkey support to your application. Also feel free to refer to [SIP-9](https://github.com/sui-foundation/sips/blob/main/sips/sip-9.md) for product specification.
+
+:::info
+Passkeys is available in beta in Sui Devnet and Testnet. The Mainnet release is yet to be scheduled.  
+::: 
+
+## Benefits of using passkeys
+
+**Sign transactions seamlessly**
+
+Users can sign transactions in Sui using passkeys, where the passkey private key stays securely stored within the authenticator, reducing the risk of key extraction attacks.
+
+**Authenticate across devices**
+
+Users can approve transactions on their mobile phones by scanning a QR code from a desktop browser. Cloud-synchronized passkeys (such as those stored in Apple iCloud or Google Password Manager) let users authenticate across multiple devices without manual key transfers.
+
+**Use hardware security keys**
+
+Users can sign transactions with external security keys, such as YubiKeys, to add an extra layer of protection against phishing and unauthorized access.
+
+**Authenticate with platform-based security**
+
+Users can sign transactions directly on devices with built-in authenticators (such as Face ID on iPhones or Windows Hello on Windows PCs). This approach lets users sign transactions natively without needing an external security key.
+
+**Recover access and secure accounts with multi-signature authentication**
```

Comment:
the cloud synchronization has nothing to do with multisig, just mention the passkey is recoverable if synced in cloud is sufficient.